core/trino-main/src/test/java/io/trino/sql/planner/TestInsert.java

```diff
@@ -99,7 +99,7 @@ public void testInsertWithPreferredPartitioning()
     {
         assertDistributedPlan(
                 "INSERT into test_table_preferred_partitioning VALUES (1, 2)",
-                withPreferredPartitioning(),
+                withPreferredPartitioning(defaultSession()),
```

Comment:
This adds unnecessary complexity to building of session. IMO previous code was cleaner